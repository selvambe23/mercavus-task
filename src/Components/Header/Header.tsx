import React from 'react';

const Header = () => (
  <div className="header">
    <div className="header-content">
        <img
          src="https://cdn.shopify.com/s/files/1/0003/1938/9747/files/Mercavus-Logo-White_1024x1024.png?v=1539856605"
          alt="logo"
        />
    </div>
  </div>
);

export default Header;
