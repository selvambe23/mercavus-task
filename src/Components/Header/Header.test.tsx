import * as React from 'react';
import { shallow } from 'enzyme';
import * as renderer from "react-test-renderer";
import Header from './Header';

describe('Header Component', () => {
  it('renders snapshots, too', () => {
    const wrapper = shallow(<Header/>);
    expect(wrapper.find('.header')).toBeDefined();

    const tree = renderer
      .create(<Header />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
});