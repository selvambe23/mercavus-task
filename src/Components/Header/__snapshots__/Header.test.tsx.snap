// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders snapshots, too 1`] = `
<div
  className="header"
>
  <div
    className="header-content"
  >
    <img
      alt="logo"
      src="https://cdn.shopify.com/s/files/1/0003/1938/9747/files/Mercavus-Logo-White_1024x1024.png?v=1539856605"
    />
  </div>
</div>
`;
